functions-and-operators/string-functions.md

```diff
@@ -1188,12 +1188,61 @@ Return a string containing octal representation of a number.
 
 ### [`OCTET_LENGTH()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_octet-length)
 
-Synonym for `LENGTH()`.
+Synonym for [`LENGTH()`](#length).
 
 ### [`ORD()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_ord)
```

Comment:
Side note, should we still point to the MySQL documentation, when we have our own description?

Reply:
As I said in https://github.com/pingcap/docs/issues/15338#issuecomment-2039767839 I think that should be done at a later stage after our own docs have been tested a bit. Then eventually we can move the links to the MySQL Reference Manual to the "See also" section.